tests/fibgo/connection_test.go

```diff
@@ -163,7 +167,16 @@ func setupValidators(coordinatorKey string, followerKey string, t *testing.T) er
 	manager2.AddListener(validatorListener2)
 
 	go func() {
-		err := rollupvalidator.LaunchRPC(manager2, "1235")
+		validatorServer := rollupvalidator.NewRPCServer(manager2, time.Second*60)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line is 81 characters, out of limit 80](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)